forge-gui/res/cardsfolder/upcoming/case_the_joint.txt

```diff
@@ -0,0 +1,6 @@
+Name:Case the Joint
+ManaCost:3 U
+Types:Instant
+A:SP$ Draw | NumCards$ 2 | StackDescription$ {p:You} draws two cards, | SubAbility$ DBDig | SpellDescription$ Draw two cards, then look at the top card of each player's library.
+SVar:DBDig:DB$ Dig | Defined$ Player | DigNum$ 1 | DestinationZone$ Library | LibraryPosition$ 0 | StackDescription$ then looks at the top card of each player's library.
```

Comment:
I think this should use `DB$ PeekAndReveal` with `NoReveal$` instead of Dig
I don't like a card this simple needing custom `StackDescription$`
Once we switch it to PeekAndReveal, if the StackDesc is wonky, I'll fix it in PeekAndReveal